src/api/app/components/add_review_collapsible_component.html.haml

```diff
@@ -0,0 +1,22 @@
+.card.collapse.position-absolute.w-100.mt-2.shadow-lg{ id: 'review_form_collapse' }
+  .card-body
+    %h5
+      Give a review for
+      %i
+    = form_tag(request_modify_review_path) do
+      = hidden_field_tag('review_id')
+      .form-group
+        = text_area_tag('comment', '', rows: 4, class: 'w-100 form-control', placeholder: 'Please comment on your decision')
+      .form-group
+        .custom-control.custom-radio
+          = radio_button_tag('new_state', 'Decline', true, class: 'custom-control-input', aria: { describedby: 'decline-description' })
+          = label_tag('Decline', 'Decline', for: 'new_state_Decline', class: 'custom-control-label')
+          %small.form-text.text-muted{ id: 'decline-description' }
```

Comment:
Let's use the short syntax for IDs. I'm a bit surprised this isn't caught by haml_lint:
```suggestion
          %small.form-text.text-muted#decline-description
```